documentation/source/coroutines.rst

```diff
@@ -144,13 +141,16 @@ forcing their completion before they would naturally end.
 .. versionchanged:: 1.4
     The :any:`cocotb.coroutine` decorator is no longer necessary for ``async def`` coroutines.
     ``async def`` coroutines can be used, without the ``@cocotb.coroutine`` decorator, wherever decorated coroutines are accepted,
-    including :keyword:`yield` statements and :func:`cocotb.fork`.
+    including :keyword:`yield` statements and ``cocotb.fork`` (since replaced with :func:`~cocotb.start` and :func:`~cocotb.start_soon`).
 
 .. versionchanged:: 1.6
     Added :func:`cocotb.start` and :func:`cocotb.start_soon` scheduling functions.
 
 .. versionchanged:: 1.7
-    Deprecated :func:`cocotb.fork`.
+    Deprecated ``cocotb.fork``.
```

Comment:
I guess we'll need to have a deeper discussion about those versionchanged statements, but I could see us removing all 1.x ones going forward, otherwise the documentation turns into a distributed changelog at some point.

I added this point to the maintainers meeting agenda.

Reply:
We didn't get to it in the meeting. Just keep what you did, we can do a global cleanup once we know what we want to do. (And absolutely don't block this PR on it.)